source/features/keyboard-navigation.tsx

```diff
@@ -0,0 +1,53 @@
+import select from 'select-dom';
+import * as pageDetect from 'github-url-detection';
+
+import features from '.';
+import {isEditable} from '../helpers/dom-utils';
+
+function runShortcuts(event: KeyboardEvent): void {
+	if (isEditable(event.target)) {
+		return;
+	}
+
+	const focusedComment = select(':target')!;
+
+	if (['j', 'k'].includes(event.key)) {
+		event.preventDefault();
+
+		const items = select.all('.js-targetable-element')
+			.filter(comment => !comment.querySelector('.minimized-comment:not(.d-none)'));
+		// `j` goes to the next comment, `k` goes back a comment
+		const direction = event.key === 'j' ? 1 : -1;
+
+		const currentIndex = items.indexOf(focusedComment);
+
+		// Start at 0 if nothing is; clamp index
+		const chosenCommentIndex = Math.min(
+			Math.max(0, currentIndex + direction),
+			items.length - 1
+		);
+
+		// Focus comment without pushing to history
+		location.replace('#' + items[chosenCommentIndex].id);
```

Comment:
I'd avoid doing this if `location.hash` is already the same as comment ID. I'm seeing some weird behavior as mentioned in https://github.com/sindresorhus/refined-github/issues/3263#issuecomment-647483078 and https://github.com/sindresorhus/refined-github/issues/3263#issuecomment-647498245.

Reply:
> if `location.hash` is already the same as comment ID

That shouldn't happen... unless there's a single file or comment on the page.

Reply:
If you are already on the first comment and press <kbd>k</kbd> again, this happens. The same goes for pressing <kbd>j</kbd> when already on the last comment.

Reply:
Indeed. I was thinking that this cycles the list. 

Let’s add a `if chosenCommentIndex !== currentIndex`